src/server/worldserver/worldserver.conf.dist

```diff
@@ -2675,6 +2675,29 @@ Battleground.TrackDeserters.Enable = 1
 
 Battleground.InvitationType = 0
 
+#
+#     Battleground.RewardWinnerHonorFirst
+#     Battleground.RewardWinnerArenaFirst
+#     Battleground.RewardWinnerHonorLast
+#     Battleground.RewardWinnerArenaLast
+#     Battleground.RewardLoserHonorFirst
```

Comment:
What does it mean "rewardloserhonorfirst" ?? what is it for?

Reply:
Reward Loser Honor First - https://imgur.com/a/jpwE8fd
Reward Loser Honor Last - https://imgur.com/a/XlsBET3

Reply:
ok :P that's a lot of points for losing :P

Reply:
because i set
Battleground.Reward.Loser.Honor.First  = 5
to
Battleground.Reward.Loser.Honor.First  =  5000 :P